### PR TITLE
re-add port check

### LINKED ISSUE #865: Access web-ui via SSH-Tunnel
Hi,

I'm accessing port 80 of my ESP32 via a SSH-Tunnel and get some errors, because using e.g. localhost:3123 makes the UI-JavaScript add your Test-Systems-IP to the URLs.
There was already a fix merged in #410 and I guess it was commented out by accident.

### PR DESCRIPTION
**Pull request by meisterlampe**
Remove comment to re-enable access via ssh-tunnels ( see #410  )
closes #865

### CODE REVIEW

**Comment by jomjol** [comment]
The problem is, that I use this localport for my local debugging of the html page. If on my PC "localhost" is detected, I switch the communication to a IP adress, that is one of my ESP32s. This way I can debug the html page on a normal PC.

There are other ways to handle this, but currently I don't want to change it.

File: sd-card/html/gethost.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function gethost_Version(){
 function getbasepath(){
     var host = window.location.hostname;
     if (((host == "127.0.0.1") || (host == "localhost") || (host == "")) 
-//       && ((window.location.port == "80") || (window.location.port == ""))
+       && ((window.location.port == "80") || (window.location.port == ""))
```
Comment (cristianmitran) [inline sd-card/html/gethost.js:10]: what's the change here? :)

**Comment by jomjol** [comment]
No change anymore!

**Comment by meisterlampe** [comment]
The change is, that the port check is included again, which is currently commented. This enables users to access the UI via a ssh-tunnel, where the adress of the esp would be e.g. http://localhost:3123. On the other hand you should be able to access the UI on your development machine using http://localhost as before (that is the check for port 80).

**Pull request closed without merging**